src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super("circle", color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public String draw() {
```

Comment:
You should draw the figure inside the method
```suggestion
    public void draw() {
```